.github/workflows/main.yml

```diff
@@ -7,7 +7,6 @@ on:
       - v*
 
   pull_request:
-    branches: [master]
```

Comment:
Yeah; that's in a separate commit. It enables gh actions for all PRs, regardless of the target branch. Sometimes I open a PR that depends on another that's pending review/merge; so I push the branch of the first to the main repo, and have the second PR target that branch (temporarily) so that the changes of the first don't show in the second's diff. (This one depends on #458)

Reply:
Ah, makes sense :+1: